snap/snapcraft.yaml

```diff
@@ -215,18 +215,23 @@ apps:
       STAGEGATE_REGISTRY_ACL_SECRETSADMINTOKENPATH: $SNAP_DATA/secrets/edgex-consul/admin/token.json
       STAGEGATE_REGISTRY_ACL_SENTINELFILEPATH: $SNAP_DATA/consul/config/consul_acl_done
       STAGEGATE_REGISTRY_ACL_TOKENBASEDIR: $SNAP_DATA/secrets
+      ADD_REGISTRY_ACL_ROLES: edgex-sys-mgmt-agent, appservice-rules-engine
```

Comment:
Why enable these specially vs just making them enabled by default?

Reply:
these two services right now are not running in secure mode by default so that is why we add it in when running in secure mode and would require consul token in secure mode.

Reply:
I think these could be added to the default list since they are always present in secure mode.

Reply:
we could do that way, too. that could save one environment override

Reply:
IMO, it is more consistent with the usage. The environment variable is for services that we don't know have been added. These two are always there.

Reply:
good point, ok i'll change it to that default list

Reply:
both `edgex-sys-mgmt-agent` and `appservice-rules-engine` have been added into the default list